client/src/apis/index.ts

```diff
@@ -1,7 +1,12 @@
 import axios from 'axios';
+import { getUserToken } from '../shared/utils';
```

Comment:
Use module path aliases instead. Apply only to all modified lines authored by you. You can search for '../'

```suggestion
import { getUserFullName, isSignedIn } from '@/src/shared/utils';
```